app/components/shared/widgets/Widget.tsx

```
@@ -301,7 +301,7 @@ class Widget<Configuration> extends Component<WidgetProps<Configuration>, Widget
                         <ErrorMessage
                             autoHide={false}
                             error={i18n.t(
-                                'widget.renderError',
+                                'widget.unexpectedError',
                                 'Cannot render widget. Check browser console for details.'
```

Default text can be removed not to mislead developers.

It was temporarily introduced, IMHO we don't need it.

---

It has been removed 🎉